static/styles.css

```diff
@@ -1,3 +1,11 @@
+.navbar {
+  border-bottom: 1px solid #5a5a5a;
+}
+
+.container {
```

Comment:
Is this just meant for pages as a whole? If so, adding this margin to `body` instead would avoid problems with nested containers.